<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>lp_peackmeter_widget_ui</class>
 <widget class="QWidget" name="lp_peackmeter_widget_ui" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="1" colspan="2" >
    <widget class="QLabel" name="lb_val" >
     <property name="minimumSize" >
      <size>
       <width>60</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>15</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>15</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>15</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" >
    <widget class="QLabel" name="lb_clip" >
     <property name="minimumSize" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>15</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" >
    <widget class="QLabel" name="lb_vu" >
     <property name="minimumSize" >
      <size>
       <width>20</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>20</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
